packages/react-components/react-table/src/components/TableCellLayout/useTableCellLayout.ts

```diff
@@ -1,6 +1,13 @@
 import * as React from 'react';
 import { getNativeElementProps, resolveShorthand } from '@fluentui/react-utilities';
 import type { TableCellLayoutProps, TableCellLayoutState } from './TableCellLayout.types';
+import { useTableContext } from '../../contexts/tableContext';
+
+const tableAvatarSizeMap = {
+  medium: undefined,
```

Comment:
Should this be `32` instead of `undefined`? Unless table specifically doesn't care about the size in this state.
```suggestion
  medium: 32,
```

Reply:
32 is the default so I didn't think I'd need to add it, but I can

Reply:
Added in https://github.com/microsoft/fluentui/pull/24807/commits/f792ca24fbac0cac20ca2f4cfc514996be2326e9